docs/intro/clients.rst

```diff
@@ -388,12 +391,74 @@ will automatically connect to the project-linked instance—no need for
 environment variables or hard-coded credentials. Follow the :ref:`Using
 projects <ref_guide_using_projects>` guide to get started.
 
-Using ``EDGEDB_DSN``
-^^^^^^^^^^^^^^^^^^^^
+Using environment variables
+---------------------------
+
+For EdgeDB Cloud
+^^^^^^^^^^^^^^^^
 
 In production, connection information can be securely passed to the client
-library via environment variables. Most commonly, you set a value for
-``EDGEDB_DSN``.
+library via environment variables. For EdgeDB Cloud instances, the recommended
+variables to set are ``EDGEDB_INSTANCE`` and ``EDGEDB_SECRET_KEY``.
+
+Set ``EDGEDB_INSTANCE`` to ``<org-name>/<instance-name>`` where
+``<instance-name>`` is the name you set when you created the EdgeDB Cloud
+instance.
+
+If you have not yet created a secret key, you can do so in the EdgeDB Cloud UI
+or by running :ref:`ref_cli_edgedb_cloud_secretkey_create` via the CLI.
+
+.. lint-off
+
+.. note::
+
+    You may also set other environment variables like ``EDGEDB_DSN`` for EdgeDB
```

Comment:
Using the underlying instance DSN directly is not really recommended practice.  I wouldn't highlight this unless there's a specific reason to.

Reply:
I believe the reason I did it was that we had a user trying to do this when I was putting together the PR, and they couldn't find how. I don't remember what the circumstances were though exactly. I'll go ahead and remove this note.